src/pages/StudyDetail/style.ts

```diff
@@ -0,0 +1,9 @@
+import styled from '@emotion/styled';
+
+export const StudyDetailContainer = styled.div`
+  width: 600px;
+  margin: 80px auto;
```

Comment:
```suggestion
  margin: 3rem auto;
```
margin top이 많이 지정된 것 같습니다! 피그마 상에선 3rem(48px)인 것 같습니다!

Reply:
반영완료